### DISCUSSION TITLE
What are vmWare/Broadcom's plans for Concourse?

### DISCUSSION THREAD

**Discussion by DanielJonesEB** [concourse/concourse#8918] [Help & Support] [answered]
'ello folks,

It's been clear for a while that vmWare are regrettably not putting much resource into Concourse. What's the latest? Can we expect it to be a community-only thing from now on? Will it ever find its way to a foundation?

Years have gone by, and I still can't get other systems to do some things as elegantly as Concourse does 😢

**Comment by eliasp** [comment, marked as answer]
There's an ongoing discussion here: https://github.com/concourse/concourse/discussions/8920
